jest.config.js

```diff
@@ -0,0 +1,20 @@
+/** @type {import('@ts-jest/dist/types').InitialOptionsTsJest} */
+export default {
```

Comment:
copied from zkapp-cli example - this is what's causing compatibility issues, but without it we can't test the contract behavior.